df_to_vec.py

```diff
@@ -0,0 +1,242 @@
+from abc import ABC, abstractmethod
+from typing import Callable, Dict, Tuple
+
+import pandas as pd
+from gensim.models import Word2Vec
+import numpy as np
+import os
+
+from pandas import Series
+
+OUTPUT_DIRECTORY = './output/vectors'
+PARAM_DATAPOINTS_DATAFRAME = './output/ml_inputs/_ml_param.csv'
+RETURN_DATAPOINTS_DATAFRAME = './output/ml_inputs/_ml_return.csv'
+
+VEC_LENGTH = 100
+NUMBER_OF_TYPES = 1000
```

Comment:
Perhaps it would be a good idea to introduce these "configs" in a separate python file because I also use them.
Maybe in a separate PR though in order to avoid dependencies between the PRs.

Reply:
Sounds like a good idea, we can do this once our PRs have been merged.